styles/scss/core/_utilities.scss

```
@@ -122,3 +122,27 @@ $indexes: map.merge(
     }
   }
 }
+
+// Dynamic viewport
+// 
+// Following the pattern of `vh-100` and `min-vh-100` from
+// https://github.com/twbs/bootstrap/blob/v4-dev/scss/utilities/_sizing.scss
+// 
+// More info on MDN
+// https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Values/length#dynamic_viewport_units
+
+.dvh-100 {
+   // fallback for browsers that don't support dynamic viewport units
+   height: 100vh !important;
+
+   // dynamic units for modern browsers
+   height: 100dvh !important;
+}
```

Do you think using `@supports` is a good way to avoid adding comments about browser support?

```suggestion
.dvh-100 {
   height: 100vh !important;

   @supports (height: 100dvh) {
     height: 100dvh !important;
   }
}
```

---

I think it's a question of what we want the compiled CSS to look like, with your suggestion the compiled CSS would be
```css
.dvh-100 {
  height: 100vh !important;
}

@supports (height: 100dvh) {
  .dvh-100 {
    height: 100dvh !important;
  }
}
```

Which means it would minify into slightly more code

```css
.dvh-100{height:100vh!important;height:100dvh!important}
```

vs

```css
.dvh-100{height:100vh!important;}
@supports (height:100dvh){.dvh-100{height:100dvh!important;}}
```

If the issue is just having comments in there, then I'd be happy to just remove the comments and have

```suggestion
.dvh-100 {
   height: 100vh !important;
   height: 100dvh !important;
}
```

as anyone familiar with `dvh` will understand that the `vh` units are there as a fallback for browsers that don't have `dvh` support, and anyone unfamiliar would be able to intuit it after searching for `dvh`.

---

I would still like to keep at least some comments if you’re not fond of using `@supports`.